*** tests/A-07-06-M-ok.out	2019-11-20 15:11:31.000000001 +0000
--- tests/A-07-06-M-ok.outhyp	2019-12-04 15:17:41.000000001 +0000
***************
*** 40,46 ****
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
--- 40,46 ----
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
***************
*** 49,52 ****
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 49,52 ----
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.po.io.RuntimeEOFException: Fim do fluxo de dados de entrada
