*** tests/A-07-11-M-ok.out	2019-11-20 15:11:31.000000001 +0000
--- tests/A-07-11-M-ok.outhyp	2019-12-06 17:56:13.000000001 +0000
***************
*** 40,58 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: ENTREGA: 1 - 1 de  3 - Livro - Analise Matematica sem Mestre - 19 - Técnica e Científica - Carlos Fonseca - 1234567890
! Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: ENTREGA: 7 - 1 de  1 - DVD - The rise of Skywalker - 20 - Técnica e Científica - J. J. Abrams - 200505551
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
--- 40,67 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 1 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
! 3 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
! 0 - Obi-Wan Kenobi - obiwan@jedi.org - NORMAL - ACTIVO
! 2 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
+ 4 - Mostrar notificações de utente
+ 5 - Saldar multa de utente
+ 0 - Sair
+ Escolha uma opção: Menu de Gestão de Utentes
+ 1 - Registar utente
+ 2 - Mostrar utente
+ 3 - Listar utentes
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
