*** tests/A-11-03-M-ok.out	2019-11-20 15:13:04.000000001 +0000
--- tests/A-11-03-M-ok.outhyp	2019-12-04 15:17:48.000000001 +0000
***************
*** 11,17 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 7 deve ser devolvida no dia 3.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 11,25 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Opção inválida!
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
