*** tests/A-11-04-M-ok.out	2019-11-20 15:13:04.000000001 +0000
--- tests/A-11-04-M-ok.outhyp	2019-12-05 10:57:34.000000001 +0000
***************
*** 11,22 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 6 deve ser devolvida no dia 8.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 8.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 11,22 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 6 deve ser devolvida no dia 0.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 0.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
