*** tests/A-11-02-M-ok.out	2019-11-20 15:13:04.000000001 +0000
--- tests/A-11-02-M-ok.outhyp	2019-12-05 10:57:34.000000001 +0000
***************
*** 16,22 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Requisitar obra: Operação inválida: O utente -1 não existe.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 16,22 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 2 deve ser devolvida no dia 0.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
