*** tests/A-12-05-M-ok.out	2019-11-20 15:13:18.000000001 +0000
--- tests/A-12-05-M-ok.outhyp	2019-12-06 10:23:53.000000001 +0000
***************
*** 7,13 ****
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
--- 7,14 ----
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir dados: Operação inválida: O ficheiro 'faltoso' não existe.
! Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
***************
*** 20,91 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 22.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 28.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Data actual: 20
! Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Introduza número de dias a avançar: Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Data actual: 22
! Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
  0 - Sair
  Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Introduza número de dias a avançar: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
--- 21,69 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Requisitar obra: Operação inválida: O utente 1 não existe.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Introduza o número de utente: Introduza o número da obra: Requisitar obra: Operação inválida: O utente 0 não existe.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Opção inválida!
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
  0 - Sair
! Escolha uma opção: Opção inválida!
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
  0 - Sair
! Escolha uma opção: Opção inválida!
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
  0 - Sair
  Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Introduza o número de utente: Introduza o número da obra: Introduza o número de utente: Introduza o número da obra: Requisitar obra: Operação inválida: O utente 3 não existe.
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
***************
*** 94,121 ****
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Data actual: 28
  Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
- 4 - Avançar data
- 5 - Menu de gestão de utentes
- 6 - Menu de gestão de obras
- 7 - Menu de gestão de requisições
- 0 - Sair
- Escolha uma opção: Menu de Gestão de Requisições
- 1 - Requisitar obra
- 2 - Devolver obra
- 0 - Sair
- Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Menu de Gestão de Requisições
- 1 - Requisitar obra
- 2 - Devolver obra
- 0 - Sair
- Escolha uma opção: Menu Principal
- 1 - Abrir dados
- 2 - Guardar dados
- 3 - Ver data
  4 - Avançar data
  5 - Menu de gestão de utentes
  6 - Menu de gestão de obras
--- 72,82 ----
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir dados: Operação inválida: O ficheiro '0' não existe.
  Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
  4 - Avançar data
  5 - Menu de gestão de utentes
  6 - Menu de gestão de obras
