*** tests/A-12-01-M-ok.out	2019-11-20 15:13:18.000000001 +0000
--- tests/A-12-01-M-ok.outhyp	2019-12-05 10:57:36.000000001 +0000
***************
*** 11,22 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Devolver obra: Operação inválida: A obra 11 não existe.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Devolver obra: Operação inválida: A obra 1 não foi requisitada pelo utente 1.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 11,26 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Requisitar obra: Operação inválida: O utente 11 não existe.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 0 deve ser devolvida no dia 0.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
***************
*** 30,33 ****
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 34,37 ----
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.po.io.RuntimeEOFException: Fim do fluxo de dados de entrada
