*** tests/A-12-05-M-ok.out	2019-11-20 15:13:18.000000001 +0000
--- tests/A-12-05-M-ok.outhyp	2019-12-06 21:48:10.000000001 +0000
***************
*** 20,124 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 22.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 28.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
! 0 - Sair
! Escolha uma opção: Data actual: 20
! Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
! 0 - Sair
! Escolha uma opção: Introduza número de dias a avançar: Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
! 0 - Sair
! Escolha uma opção: Data actual: 22
! Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
! 0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
! 0 - Sair
! Escolha uma opção: Introduza número de dias a avançar: Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
! 0 - Sair
! Escolha uma opção: Data actual: 28
! Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 20,63 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Requisitar obra: Operação inválida: O utente 1 não pode requisitar a obra 1. Violação da regra 2.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Introduza o número de utente: Introduza o número da obra: A obra 3 deve ser devolvida no dia 35.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Opção inválida!
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Devolver obra: Operação inválida: O utente 3 não existe.
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Introduza o número de utente: Introduza o número da obra: Devolver obra: Operação inválida: A obra 4 não foi requisitada pelo utente 0.
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
  0 - Sair
! Escolha uma opção: Opção inválida!
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Opção inválida!
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Opção inválida!
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Introduza o número de utente: Introduza o número da obra: Introduza o número de utente: Fim de entrada de dados (R-EOF): pt.tecnico.po.io.RuntimeEOFException: Fim do fluxo de dados de entrada
