*** tests/A-13-01-M-ok.out	2019-11-20 15:13:26.000000001 +0000
--- tests/A-13-01-M-ok.outhyp	2019-12-04 15:17:51.000000001 +0000
***************
*** 20,32 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 22.
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 2 deve ser devolvida no dia 35.
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
--- 20,46 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
***************
*** 62,77 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: O utente 1 deve pagar uma multa de EUR 90.
! O utente deseja pagar multa (s/n)? Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: O utente 2 deve pagar uma multa de EUR 25.
! O utente deseja pagar multa (s/n)? Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
--- 76,107 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
+ Escolha uma opção: Número inválido!
+ Escolha uma opção: Menu de Gestão de Requisições
+ 1 - Requisitar obra
+ 2 - Devolver obra
+ 0 - Sair
+ Escolha uma opção: Menu de Gestão de Requisições
+ 1 - Requisitar obra
+ 2 - Devolver obra
+ 0 - Sair
+ Escolha uma opção: Menu de Gestão de Requisições
+ 1 - Requisitar obra
+ 2 - Devolver obra
+ 0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
***************
*** 89,96 ****
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - SUSPENSO - EUR 90
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - SUSPENSO - EUR 25
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 119,126 ----
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
***************
*** 98,113 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - SUSPENSO - EUR 25
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 128,141 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Novo utente criado com o número 3.
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
***************
*** 115,130 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Menu de Gestão de Utentes
! 1 - Registar utente
! 2 - Mostrar utente
! 3 - Listar utentes
! 4 - Mostrar notificações de utente
! 5 - Saldar multa de utente
! 0 - Sair
! Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 143,149 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: 3 - 3 - 5 - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
