*** tests/A-13-05-M-ok.out	2019-11-20 15:13:26.000000001 +0000
--- tests/A-13-05-M-ok.outhyp	2019-12-06 15:27:17.000000001 +0000
***************
*** 14,28 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Saldar multa de utente: Operação inválida: O utente 1 não se encontra suspenso.
! Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Saldar multa de utente: Operação inválida: O utente 10 não existe.
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 14,27 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Novo utente criado com o número 4.
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
