*** tests/A-03-03-M-ok.out	2019-11-20 15:10:42.000000001 +0000
--- tests/A-03-03-M-ok.outhyp	2019-12-03 11:54:12.000000001 +0000
***************
*** 12,23 ****
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
! Escolha uma opção: Introduza o termo de pesquisa: 1 - 3 de  3 - Livro - Analise Matematica sem Mestre - 19 - Técnica e Científica - Carlos Fonseca - 1234567890
! Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
--- 12,23 ----
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
! Escolha uma opção: Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
