*** tests/A-05-04-M-ok.out	2019-11-20 15:11:09.000000001 +0000
--- tests/A-05-04-M-ok.outhyp	2019-12-03 11:54:16.000000001 +0000
***************
*** 14,29 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 2 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 5 - Chewbacca - chewbacca2@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
  4 - Darth Vader - darth2@dark.force.coAm - NORMAL - ACTIVO
! 7 - Han Solo - han.solo@milenium.falcon.org - NORMAL - ACTIVO
  9 - Han Solo - han.solo2@milenium.falcon.org - NORMAL - ACTIVO
! 0 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Luke Skywalker - last.jedi2@universe.com - NORMAL - ACTIVO
! 6 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  8 - Rey - real.last.jedi2@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 14,29 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 5 - Chewbacca - chewbacca2@milenium.falcon.org - NORMAL - ACTIVO
! 2 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
  4 - Darth Vader - darth2@dark.force.coAm - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
  9 - Han Solo - han.solo2@milenium.falcon.org - NORMAL - ACTIVO
! 7 - Han Solo - han.solo@milenium.falcon.org - NORMAL - ACTIVO
  3 - Luke Skywalker - last.jedi2@universe.com - NORMAL - ACTIVO
! 0 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  8 - Rey - real.last.jedi2@universe.com - NORMAL - ACTIVO
+ 6 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
