*** tests/A-07-10-M-ok.out	2019-11-20 15:11:31.000000001 +0000
--- tests/A-07-10-M-ok.outhyp	2019-12-04 14:11:29.000000001 +0000
***************
*** 11,45 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 7 deve ser devolvida no dia 3.
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Deseja ser avisado quando algum exemplar for devolvido (s/n)? Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 8.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 8.
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 8.
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Deseja ser avisado quando algum exemplar for devolvido (s/n)? Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
--- 11,60 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Opção inválida!
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
! 0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
***************
*** 52,57 ****
--- 67,82 ----
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
+ Escolha uma opção: Ficheiro a abrir: Abrir dados: Operação inválida: O ficheiro '1' não existe.
+ Menu Principal
+ 1 - Abrir dados
+ 2 - Guardar dados
+ 3 - Ver data
+ 4 - Avançar data
+ 5 - Menu de gestão de utentes
+ 6 - Menu de gestão de obras
+ 7 - Menu de gestão de requisições
+ 0 - Sair
  Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
***************
*** 61,67 ****
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
--- 86,93 ----
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir dados: Operação inválida: O ficheiro '3' não existe.
! Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
***************
*** 70,94 ****
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Menu de Gestão de Utentes
! 1 - Registar utente
! 2 - Mostrar utente
! 3 - Listar utentes
! 4 - Mostrar notificações de utente
! 5 - Saldar multa de utente
! 0 - Sair
! Escolha uma opção: Introduza o número de utente: ENTREGA: 7 - 1 de  1 - DVD - The rise of Skywalker - 20 - Técnica e Científica - J. J. Abrams - 200505551
! Menu de Gestão de Utentes
! 1 - Registar utente
! 2 - Mostrar utente
! 3 - Listar utentes
! 4 - Mostrar notificações de utente
! 5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
  4 - Avançar data
  5 - Menu de gestão de utentes
  6 - Menu de gestão de obras
--- 96,125 ----
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir dados: Operação inválida: O ficheiro 's' não existe.
! Menu Principal
! 1 - Abrir dados
! 2 - Guardar dados
! 3 - Ver data
! 4 - Avançar data
! 5 - Menu de gestão de utentes
! 6 - Menu de gestão de obras
! 7 - Menu de gestão de requisições
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
  3 - Ver data
+ 4 - Avançar data
+ 5 - Menu de gestão de utentes
+ 6 - Menu de gestão de obras
+ 7 - Menu de gestão de requisições
+ 0 - Sair
+ Escolha uma opção: Ficheiro a abrir: Abrir dados: Operação inválida: O ficheiro '7' não existe.
+ Menu Principal
+ 1 - Abrir dados
+ 2 - Guardar dados
+ 3 - Ver data
  4 - Avançar data
  5 - Menu de gestão de utentes
  6 - Menu de gestão de obras
