*** tests/A-07-08-M-ok.out	2019-11-20 15:11:31.000000001 +0000
--- tests/A-07-08-M-ok.outhyp	2019-12-04 14:11:28.000000001 +0000
***************
*** 11,26 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 7 deve ser devolvida no dia 3.
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 2 deve ser devolvida no dia 15.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
--- 11,33 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Opção inválida!
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
+ Escolha uma opção: Menu de Gestão de Requisições
+ 1 - Requisitar obra
+ 2 - Devolver obra
+ 0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
***************
*** 39,42 ****
--- 46,59 ----
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
+ Escolha uma opção: Menu Principal
+ 1 - Abrir dados
+ 2 - Guardar dados
+ 3 - Ver data
+ 4 - Avançar data
+ 5 - Menu de gestão de utentes
+ 6 - Menu de gestão de obras
+ 7 - Menu de gestão de requisições
+ 0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: 
\ No newline at end of file
