*** tests/A-08-05-M-ok.out	2019-11-20 15:11:40.000000001 +0000
--- tests/A-08-05-M-ok.outhyp	2019-12-04 14:11:31.000000001 +0000
***************
*** 14,20 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Registar utente: Operação inválida: User registration failed: name 'as', email ''.
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 14,20 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Novo utente criado com o número 3.
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
***************
*** 22,28 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Novo utente criado com o número 3.
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 22,28 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Novo utente criado com o número 4.
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
***************
*** 30,39 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 0 - Darth Vader - darth@dark.force.com - NORMAL - ACTIVO
  1 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  2 - Obi-Wan Kenobi - obiwan@jedi.org - NORMAL - ACTIVO
! 3 - Y - 3 - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 30,40 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 3 - as -  - NORMAL - ACTIVO
! 0 - Darth Vader - darth@dark.force.com - NORMAL - ACTIVO
  1 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  2 - Obi-Wan Kenobi - obiwan@jedi.org - NORMAL - ACTIVO
! 4 - Y - 3 - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
