*** tests/A-11-01-M-ok.out	2019-11-20 15:13:04.000000001 +0000
--- tests/A-11-01-M-ok.outhyp	2019-12-04 14:11:32.000000001 +0000
***************
*** 27,33 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Requisitar obra: Operação inválida: A obra 10 não existe.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 27,41 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Opção inválida!
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
