*** tests/A-12-01-M-ok.out	2019-11-20 15:13:18.000000001 +0000
--- tests/A-12-01-M-ok.outhyp	2019-12-03 11:54:36.000000001 +0000
***************
*** 11,26 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Devolver obra: Operação inválida: A obra 11 não existe.
! Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: Devolver obra: Operação inválida: A obra 1 não foi requisitada pelo utente 1.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
--- 11,41 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Opção inválida!
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
+ Escolha uma opção: Menu de Gestão de Requisições
+ 1 - Requisitar obra
+ 2 - Devolver obra
+ 0 - Sair
+ Escolha uma opção: Menu de Gestão de Requisições
+ 1 - Requisitar obra
+ 2 - Devolver obra
+ 0 - Sair
+ Escolha uma opção: Menu de Gestão de Requisições
+ 1 - Requisitar obra
+ 2 - Devolver obra
+ 0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir dados
  2 - Guardar dados
